airgun/views/task.py

```diff
@@ -14,18 +21,51 @@ class TaskReadOnlyEntry(ReadOnlyEntry):
 
 class TasksView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Tasks']")
+    focus = ActionsDropdown(
+        "//div[./button[@id='tasks-dashboard-time-period-dropdown']]"
+    )
     table = SatTable(
         ".//div[@id='tasks-table']/table",
         column_widgets={
             'Action': Text('./a'),
         }
     )
+    table_row_count = Text("//span[@class='pagination-pf-items-total']")
 
     @property
     def is_displayed(self):
         return self.browser.wait_for_element(
             self.title, exception=False) is not None
 
+    @View.nested
+    class RunningChart(View):
+        ROOT = ".//div[@id='running-tasks-card']"
+        name = Text("./h2")
+        total = PieChart("./div[@class='card-pf-body']")
+
+    @View.nested
+    class PausedChart(View):
+        ROOT = ".//div[@id='paused-tasks-card']"
+        name = Text("./h2")
+        total = PieChart("./div[@class='card-pf-body']")
+
+    @View.nested
+    class StoppedChart(View):
+        ROOT = ".//div[@id='stopped-tasks-card']"
+        name = Text("./h2")
+        table = Table(
+            locator='.//table',
+            column_widgets={
+                'Action': Text('./button'),
```

Comment:
Is that column name is correct? I see column names as Total and 24h
![Tasks](https://user-images.githubusercontent.com/3190629/65610450-fe0c8080-dfce-11e9-8add-3860967511b5.png)